client/src/vue.d.ts

```diff
@@ -1,4 +1,5 @@
 import Vue from 'vue-property-decorator'
+import quill from 'quill'
```

Comment:
I don't see where this is used in here...?

Reply:
its not... will remove

Reply:
> 1. Are there arrows in the gray boxes that need to be changed to "to"?
needs to be changed in backend
> 2. Where is the text (option.header) that goes beside the "Option x:" title? Does it match the InVision design?
It will when the text in the backend gets updated
> 3. Where is the change from "Helpful Tip" to "Required Action"?
backend change
> 4. Where is the change to the text in the Obtain Consent box? (With checkbox and with button.)
backend change
> 5. Where is the change to the text in the Add Words box?
> 6. Where is the change to the content in the Send for Review box?
ditto
> 7. Where is the change to the Change Business Type box?
> 8. Where is the "Move the designation to the end of the name" text change?
same
> 9. Where is the "Review your name as it appears..." text/content change?
likewise

Reply:
its not.  removed.